dashboard/test/ui/features/star_labs/can_see_finish.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Make sure we can see the finish button for all LEVEL TYPE levels on small screens
+
+  @no_mobile
+  Scenario: Can see finish button for free play "Dance Party", "Artist", "Bounce" on small screens
+    Given I create an authorized teacher-associated student named "Sally"
+    And I sign in as "Teacher_Sally" and go home
+
+    And I check that the blockly free play level for "Dance Party" shows the finish button for small screens
+    And I check that the blockly free play level for "Artist" shows the finish button for small screens
+    And I check that the blockly free play level for "Bounce" shows the finish button for small screens
+    And I check that the blockly free play level for "CS in Algebra" shows the finish button for small screens
+    And I check that the blockly free play level for "Flappy" shows the finish button for small screens
+    And I check that the blockly free play level for "Sprite Lab" shows the finish button for small screens
+
+    And I check that the droplet free play level for "Game Lab" shows the finish button for small screens
+
+    And I check that the minecraft free play level for "Minecraft Adventurer" shows the finish button for small screens
+
+    # The following levels currently do not render finish button for small screens.
```

Comment:
By "do not render" do you mean that it's still available but the user has to scroll it into view?  "Not visible" capture the lesser severity of this situation :)

Reply:
Is this also a sign that we should use slightly different responsive values to make sure the button is visible?

Reply:
Yes - I will make that more explicit, thanks for the catch

Reply:
I am not sure what the fix would be - it may depend on the specific app. For instance, in minecraft levels there is an additional row of buttons we do not have in the other level types that pushes the finish button out of the viewport. We are currently planning on making the fix more general in a future change/task, since there is one unrelated task we'd like to do before working on this more thoroughly (due to a deadline).